include/RAJA/exec-cuda/reduce_cuda.hxx

```diff
@@ -1032,7 +1037,7 @@ class ReduceMinLoc<cuda_reduce<BLOCK_SIZE>, T> {
           RAJA_MINLOC(sd[threadId],
                       m_blockdata[m_blockoffset + blockId + 1]);
       __threadfence();
-      int oldBlockCount = atomicAdd(&retiredBlocks[m_myID], (int)1);
+      unsigned int oldBlockCount = ::atomicAdd((unsigned int*)&retiredBlocks[m_myID], (unsigned int)1);
```

Comment:
I realize it might be confusing, but the atomicAdd for retired blocks which is unsigned int caused the compiler to complain that it could not disambiguate for atomicAdd(double *, double) -- go figure. But, this global namespace qualifier was all it needed.